charts/incubator/watcharr/values.yaml

```diff
@@ -0,0 +1,45 @@
+image:
+  repository: tccr.io/truecharts/watcharr
+  pullPolicy: IfNotPresent
+  tag: v1.16.0@sha256:bfd121ce8ba69b9eebfcb13d1a2adad392bd0d1adca7bbca6b31c2b295d9862e
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 3080
+        port: 10684
+
+watcharr:
+  mode: prod
+  jellyfin_host: ""
+  signup_enabled: true
+  tmdb_key: ""
+  debug: false
+
+persistence:
+  data:
+    enabled: true
+    mountPath: "/data"
+  watcharr-env:
+    enabled: true
+    type: configmap
+    objectName: watcharr-env
+    targetSelector:
+      main:
+        main:
+          mountPath: /.env
+          subPath: .env
+          readOnly: true
```

Comment:
Anything that can load variables from `.env` **should** be able to read actual env variables too!

Reply:
its coded to read the file and not the variables.

Reply:
![image](https://github.com/truecharts/charts/assets/47820033/af272556-1dc3-4a65-b9d6-d185f0d9bd72)

Worst case, if it fails to load the `.env` mount an empty .env file

Reply:
https://github.com/sbondCo/Watcharr/pull/131

Reply:
> [sbondCo/Watcharr#131](https://github.com/sbondCo/Watcharr/pull/131)

Thanks again for the contribution! Took me longer than I thought, but `v1.16.1` is now available with the fix you made.

https://github.com/sbondCo/Watcharr/releases/tag/v1.16.1